README.md

```diff
@@ -261,6 +261,121 @@ the static calls to resources will run successfully. If the Client is directly
 instantiated (using the ClientBuilder or the Client constructor), the Client
 instance must be used to start interactions with Stormpath.
 
+### Cache
+By default all items will be cached.  This will use Shared Memory caching if nothing is defined.
```

Comment:
Yes, It was..  This has been removed in latest commit

On Wed Jan 28 2015 at 1:39:42 PM ecrisostomo <notifications@github.com>
wrote:

> In README.md
> <https://github.com/stormpath/stormpath-sdk-php/pull/18#discussion_r23710622>
> :
>
> > @@ -261,6 +261,121 @@ the static calls to resources will run successfully. If the Client is directly
> >  instantiated (using the ClientBuilder or the Client constructor), the Client
> >  instance must be used to start interactions with Stormpath.
> >
> > +### Cache
> > +By default all items will be cached.  This will use Shared Memory caching if nothing is defined.
>
> The Shared Memory ghost?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stormpath/stormpath-sdk-php/pull/18/files#r23710622>.
>